packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -87,6 +87,14 @@ export type BaseInputProps = FormInputLabelProps &
      * For React Native this will call `onEndEditing` event since we want to get the last value of the input field
      */
     onBlur?: FormInputOnEvent;
+    /**
+     * Ignores the next blur event
+     */
+    shouldIgnoreBlurAnimation?: boolean;
+    /**
+     * Used to set the shouldIgnoreBlurAnimation flag back to false after blur is ignored
+     */
+    setShouldIgnoreBlurAnimation?: (shouldIgnoreBlurAnimation: boolean) => void;
```

Comment:
Can we contain the setter logic within `BaseInput` to ignore the blur animations based on a flag? Right now we're passing this flag from outside and also a setter for it.

Reply:
we actually have to handle it on the outside because the flag setting logic happens when ActionListItem is clicked.

We're only passing prop here so that we can toggle the flag once one event is ignored.